# Bentley Group

## About Us

We work with whole genome sequence data to understand how bacterial pathogens colonise, get transmitted and, in some cases, cause disease.

This can help us understand how pathogens evade clinical intervention strategies (eg. antibiotics and vaccines) and how these differences can be exploited to track the spread of pathogens thus aiding the surveillance effort to medical, scientific and public health efforts to tackle these diseases.


## Research Areas (Christine - Ste to refine)

Infectious disease is one of the leading causes of mortality in children under five [UN-IGME report 2019](https://childmortality.org/reports). ...The advent of large-scale genomic sequencing technology, open access data and increasing collaborations worldwide has enabled  research scientists to tackle the problem of infectious disease from a global standpoint. ... Through various projects, we aim to understand the evolution of bacteria associated with respiratory infections in a global context. In particular are those whose major disease burden are in resource-poor settings. By understanding how genetics contributes to the survival and spread of disease causing bacteria, we can provide possible solutions to curtail the advancement of these bacteria that continue to evade human interventions such as vaccines and antibiotics.

Our key species of interest include:

### Streptococcus pneumoniae
[GPS](https://www.pneumogen.net/)

The Global Pneumococcal Sequencing (GPS) project produced the largest database of pneumococcal genomes and successfully demonstrated the importance of genomic surveillance by detecting key pneumococcal strains that evade vaccine and antibiotics on national and international level. In the GPS 2.0, we will extend the temporal coverage to characterise the post vaccine equilibrated population so as to inform future vaccine design, and span the geographical coverage to investigate the pneumococcal population in regions with high burden of pneumococcal diseases. To establish a working system for sustainable genomic surveillance, providing bioinformatic training to our project partners, capacity building in priority countries and engaging policy makers are also our key activities in the GPS2.0.

### Neisseria meningitidis
*Neisseria meningitidis*, the meningococcus, is a species of bacteria found exclusively in humans, which is a considerable cause of bacterial meningitis, as well as septicemia. These infections are difficult to treat and have a high case fatality rate, but are asymptomatic carriage infections in most cases.  The meningococcus has a substantial disease burden in the meningitis belt of sub-saharan Africa, where the incidence of infection and disease are much higher than elsewhere globally. We work on several whole-genome sequenced datasets of N. meningitidis including a large Burkina Faso carriage [collection](https://www.ncbi.nlm.nih.gov/pubmed/23914778), the MenAfriCar meningitis belt carriage surveillance project, and a global collection of carriage and disease isolates with considerable temporal spread.

### Streptococcus agalactiae
[Juno](gbsgen.net)

Group B *Streptococcus* (GBS, a.k.a. *Streptococcus agalactiae*) is an opportunistic pathogen and a major cause of neonatal invasive disease, with a particular impact on infant mortality in low and middle-income countries (LMICs), where disease prevention strategies are less often implemented or have low coverage. The JUNO project will conduct a global genomic survey of GBS isolates from across a broad geographical and temporal span with the central aim of informing vaccine development and other disease intervention strategies. The project will focus on three key areas of GBS biology: global genomic diversity, genetic associations with the disease and transmission.

## Our Software

### [FastBaps](https://github.com/gtonkinhill/fastbaps)
A fast solution to the genetic clustering problem. Fastbaps rapidly identifies an approximate fit to a Dirichlet process mixture model (DPM) for clustering multilocus genotype data. Our efficient model-based clustering approach is able to cluster datasets 10–100 times larger including over 110,000 sequences of HIV-1 pol genes.

### [Panaroo](https://github.com/gtonkinhill/panaroo)
Panaroo is a graph based pangenome clustering tool that is able to account for many of the sources of error introduced during the annotation of prokaryotic genome assemblies.

### [Pairsnp](https://github.com/gtonkinhill/pairsnp)
A set of scripts for very quickly obtaining pairwise SNP distance matrices from multiple sequence alignments. For larger alignments, pairsnp is an order of magnitude faster than approaches based on pairwise comparison of every site.

## Meet the team

### Professor Stephen D. Bentley - Team Leader
![Professor Stephen D. Bentley - Team Leader](people/headshot-small.png "Professor Stephen D. Bentley - Team Leader")
I’m interested in exploring how we can use genomics in efforts to reduce the burden of bacterial pathogens causing pneumonia, meningitis and neonatal sepsis in infants in low income settings.

### Dr Chrispin Chaguza - Post Doctoral Fellow
![Dr Chrispin Chaguza - Post Doctoral Fellow ](people/headshot-small.png "Dr Chrispin Chaguza - Post Doctoral Fellow ")
My research uses high-throughput sequencing approaches to understand the epidemiology, evolution and genetics of bacterial phenotypes primarily in clinically-significant pathogens.

### Dr Christine J. Boinett - Product Manager and Principal Bioinformatician
I am involved in developing, managing and delivering bioinformatics training for our JUNO and GPS2 project partners.

### Dr. Dorota Jamrozy - Project Manager and Principal Bioinformatician
I’m interested in exploring how we can use genomics in efforts to reduce the burden of bacterial pathogens causing pneumonia, meningitis and neonatal sepsis in infants in low income settings.

### Dr Felix Dube - Visiting Researcher
Analysis of microbial populations (with a focus on *Streptococcus pneumoniae*) to understand the epidemiology and spread of antimicrobial resistance. 

### Gareth Peat - Principal Software Developer
I am involved in developing the software and infrastructure for the JUNO project.

### Gerry Tonkin-HIll - PhD Student
I study within-host evolution and transmission in *Streptococcus pneumoniae* as well as  developing statistical and bioinformatics methods for microbial genomics.

### Neil MacAlasdair - PhD student
I study recombination and its effects on evolution in Neisseria meningitidis and how this affects various populations.

### Dr. Stephanie Wai-U Lo - Project Manager and Principal Bioinformatician 
I oversee the Global Pneumococcal Sequencing Project by setting overall scientific strategy, communicating with collaborators, and delivering outputs through publications.

### Uzma Basit Khan - Senior Computational Biologist
I study antibiotic resistance mechanisms, recombination and mobile genetic elements (MGEs) transmission in Streptococcus agalactiae to comprehend bacterial adaptation and evolution.

## Selected Publications
TODO

## Funders
- [Bill and Melinda Gates foundation](http://www.gatesfoundation.org/)
- [Meningitis Research Foundation](https://www.meningitis.org/?gclid=Cj0KCQiAnL7yBRD3ARIsAJp_oLY4izo2h6Pu4s-ENL8Us0UItFx-wpTs2-eHI5xKJu8S7ZBnNfHmi3UaAj8AEALw_wcB)
- [BBSRC](https://bbsrc.ukri.org/)
- [MRC](https://mrc.ukri.org/)
- [Wellcome](https://wellcome.ac.uk/)

## Contacts
(Please add as appropriate)
- [Stephen Bentley] (sdb@sanger.ac.uk)
- [GPS] (gps@pneumogen.net)
- [JUNO] (juno@sanger.ac.uk)
